### PR TITLE
Update: Card and Statistic Components

### PR DESCRIPTION
**Pull request by omgaz**
- Change `display:` from `inline-block` to `inline-flex`
- Make contents of card component support <FlexSpacer /> component
- Card content component now supports custom classNames
- Add new `inline` prop to statistic to have inline statistic labels and values
- Tidy up spacing
- Fix enzyme dependency error in tests
- Expose coverage report in console when tests are run

### CODE REVIEW

**Comment by etaoins** [comment]
LGTM

**Comment by mcslash** [comment]
LGTM.

**Comment by pphminions** [comment]
LGTM 🍏

File: test/components/alexandria/StatisticComponentTest.js [hunk 1]
```diff
@@ -21,4 +21,9 @@ describe('StatisticComponent', () => {
       </label>
     )).to.equal(true);
   });
+
+  it('should render inline', () => {
+    const component = shallow(<StatisticComponent label="Views" value="2 Million" inline />);
```
Comment (antklim) [inline test/components/alexandria/StatisticComponentTest.js]: Should `inline` be a boolean as it defined in component?

**Comment by mcslash** [comment]
Merged! 🚀

**Pull request merged**